.vscode/settings.json

```diff
@@ -9,5 +9,8 @@
 	"files.trimTrailingWhitespace": true,
 	"files.insertFinalNewline": true,
 	"editor.formatOnSave": true,
-	"tslint.autoFixOnSave": true
+	"tslint.autoFixOnSave": true,
+	"yaml.schemas": {
```

Comment:
I think this second line was a testing feature, you can remove the yaml.schemas: settings item for the bukkit-plugin yaml schema. This was to test acb.yaml but I guess I left it in some commit by mistake.

Reply:
I'll do that.
Thanks!